### PR TITLE
allow to run forget before pruning

### PR DESCRIPTION
**Pull request by schewara**
It should be possible to also run `restic forget ..` as part of the pruning workflow.

The reasons behind this is the following

When running the REST backend with `--append-only`, due to security restrictions, setting the `RESTIC_FORGET_ARGS` as part of the backup will fail.

Instead it should be also possible to re-use the `RESTIC_FORGET_ARGS` as part of the prune process which is then run directly on the same host as the REST backend to manage forget policies in a central place.

I am aware that the `prune` could have been combined as part of the `forget`, but for clarity I kept this as a seperate step.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ E.g.
 * `RESTIC_PASSWORD` - Password for the restic repository. Will also be used to initialize the repository if it is not yet initialized
 * `RESTIC_BACKUP_ARGS` - If specified `restic backup` is run with the given arguments, e.g. for tags, exclude definitions, or verbose logging: `--tag docker-volumes --exclude-file='exclude.txt' --verbose`. See the [restic backup documentation](https://restic.readthedocs.io/en/stable/040_backup.html) for available options
 * `RESTIC_BACKUP_TAGS` - *Deprecated*. Tags to set for each snapshot, separated by commas. This option will soon be removed. Please use `RESTIC_BACKUP_ARGS` to define tags.
-* `RESTIC_FORGET_ARGS` - If specified `restic forget` is run with the given arguments after each backup, e.g. `--prune --keep-last 14 --keep-daily 1`. See the [restic forget documentation](https://restic.readthedocs.io/en/stable/060_forget.html) for available options
+* `RESTIC_FORGET_ARGS` - If specified `restic forget` is run with the given arguments after each backup of before every prune, e.g. `--prune --keep-last 14 --keep-daily 1`. See the [restic forget documentation](https://restic.readthedocs.io/en/stable/060_forget.html) for available options
```
Comment (djmaze) [inline README.md:74]: `or`?

**Review by djmaze — CHANGES_REQUESTED** [review]
Small type, but otherwise LGTM!

File: README.md (hunk 1 above)
Comment (schewara) [reply README.md]: Thanks for spotting it. It is now corrected :heavy_check_mark:

**Comment by schewara** [comment]
@djmaze, I suppose the typo fixes never appeared on your radar, therefore a small :reminder_ribbon:

**Comment by djmaze** [comment]
Oh, sorry! Yes, looks good now. Thanks!

**Pull request merged**